src/php/globals.rs

```diff
@@ -16,6 +16,14 @@ impl ExecutorGlobals {
             .expect("Static executor globals were invalid")
     }
 
+    fn get_mut() -> &'static mut Self {
+        // SAFETY: PHP executor globals are statically declared therefore should never
+        // return an invalid pointer.
+        // TODO: Should this be syncronized?
+        unsafe { ext_php_rs_executor_globals().as_mut() }
+            .expect("Static executor globals were invalid")
```

Comment:
Kept this private because I don't know in general if this is thread safe. As I understand it, the `take_exception` function should be safe.